public/js/notes.js

```diff
@@ -20,3 +30,11 @@ function update() {
 }
 update();
 
+function deleted(itemIndex) {
```

Comment:
Function name not clear what function do.

Reply:
"delete" would be good function name here. Avoid using past and future tens words.

Reply:
delete is reserved keywords that's why I have used deleted

Reply:
You can use "deleteNote"

Reply:
Also "updateNote" instead of "update"

Reply:
okay